gunicorn/arbiter.py

```diff
@@ -535,7 +536,8 @@ def spawn_worker(self):
                 worker.tmp.close()
                 self.cfg.worker_exit(self, worker)
             except:
-                pass
+                self.log.error("Exception during worker exit:\n%s",
```

Comment:
I had a similar thought. I agree about `log.exception`. I think an error level is appropriate, though, because the application did error. Or at least warning, but maybe just error. I think debug is too low when there was an actual fault.